cli/cli/go.mod

```diff
@@ -44,14 +43,13 @@ require (
 	github.com/briandowns/spinner v1.20.0
 	github.com/fatih/color v1.13.0
 	github.com/kurtosis-tech/kurtosis-portal/api/golang v0.0.0-20230328194643-b4dea3081e25
-	github.com/kurtosis-tech/kurtosis/lsp v0.0.0-00010101000000-000000000000
+	github.com/kurtosis-tech/kurtosis/lsp v0.0.0-20230329203818-3d66efd4fce8
```

Comment:
We can drop this dependency, right? The package doesn't even exist anymore

Reply:
yeah; suprised that `go mod tidy` didn;t clean this. Let me re-run it.